doc/man3/OPENSSL_init_crypto.pod

```diff
@@ -175,10 +175,10 @@ See OPENSSL_fork_prepare(3) for details.
 
 =item OPENSSL_INIT_NO_ATEXIT
 
-By default OpenSSL will attempt to clean itself up when the process exits via an
-"atexit" handler. Using this option suppresses that behaviour. This means that
-the application will have to clean up OpenSSL explicitly using
-OPENSSL_cleanup().
+Deprecated, it's a no-op currently. Starting version 3.4 OpenSSL
```

Comment:
This is not a no-op in this PR. It will prevent `OPENSSL_cleanup()` being called from the library destructor.